src/ResourceManager/Sql/Commands.Sql/Failover Group/Model/AzureSqlFailoverGroupModel.cs

```diff
@@ -41,31 +40,11 @@ public class AzureSqlFailoverGroupModel
         public string FailoverGroupName { get; set; }
 
         /// <summary>
-        /// Gets or sets the read-write endpoint
-        /// </summary>
-        public ReadWriteEndpoint FailoverGroupReadWriteEndpoint { get; set; }
-
-        /// <summary>
-        /// Gets or sets the read-only endpoint
-        /// </summary>
-        public ReadOnlyEndpoint FailoverGroupReadOnlyEndpoint { get; set; }
-
-        /// <summary>
         /// Gets or sets the partner servers
         /// </summary>
         public IList<FailoverGroupPartnerServer> PartnerServers { get; set; }
 
         /// <summary>
-        /// Gets or sets the name of the partner resource group name
-        /// </summary>
-        public string PartnerResourceGroupName { get; set; }
```

Comment:
@ziwa-msft is this part of any return type from a cmdlet? if yes this is breaking change.